src/ucm/util/sys.c

```diff
@@ -24,16 +24,21 @@
 
 #define UCM_PROC_SELF_MAPS "/proc/self/maps"
 
+#if UCM_BISTRO_HOOKS
```

Comment:
can we define UCM_DEFAULT_HOOK_MODE and UCM_DEFAULT_HOOK_MODE_STR in same place (under same `#if`)